source/django_sourcerer/domain/import_datas/parsing_config_file.py

```diff
@@ -0,0 +1,8 @@
+import yaml
+
+
+class ReadConfigFile:
+    def import_yaml_file(self):
+        with open("csv_config_file.yaml", 'r') as stream:
```

Comment:
The filename should be a method parameter - `import_yaml_file(self, config_file)`.